apps/ejabberd/test/component_reg_SUITE.erl

```diff
@@ -10,10 +10,10 @@ all() ->
     [ registering, registering_with_local ].
 
 init_per_suite(C) ->
-    application:ensure_all_started(stringprep),
```

Comment:
Why `stringprep` is being started without `application` help all of a sudden everywhere?